### ISSUE TITLE
Component / Block headers - same experience

### ISSUE THREAD

**Issue by Tamayuzu** [MeD-DMC/Adapt-accessibilityfixes#191]
Screen reader users can hear when a component is Completed or Incomplete but users not using a screen reader will always see the normal title - For accessibility all users should be able to see the completion status of a component / block / article (same info as screen reader)

**Comment by ftourangeau** [comment]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
**Page:**
Course Overview page
Lesson 1: Overview of electronic Passport Next Generation
Lesson 2: Domestic Print
Lesson 3: Observation Print
Lesson 4: Manual QA
Lesson 5: Reports

**Description:**
Visible and accessible headings are different from each other. For example:
   • Visible heading: "Lesson 1: Overview of electronic Passport Next Generation"
   • Accessible heading: "Incomplete Lesson 1: Overview of electronic Passport Next Generation"
The "Incomplete" status will be replaced with the "Complete" one when all elements in the section are opened and/or selected. But 
1. the change is not programmatically determined through role
2. "Complete/Incomplete" status is not displayed for sighted users

1. The change of the completing status should be announced for AT users.
2. "Incomplete" information should be provided for all users (Not only for screen reader users). Nobody should be at a disadvantage.

Recommendation:
   • Visible and accessible headings should be the same. The best practice is to implement all headings using native HTML `<h>` elements (according to "First Rule of ARIA Use")
   • Use aria-live containers for heading elements
So all users will have the same information and be notified in time. For example:
`<div role="status">` (or `<div role="region" aria-live="polite">`)
   `<h1>`Complete - Lesson 1: Overview of electronic Passport Next Generation`</h1>`
`</div>`
![image](https://user-images.githubusercontent.com/93537804/229921815-54ad685f-7609-4092-aba9-08b9e8824dd3.png)
*******************************************************************************
